common/app/layout/WidthsByBreakpoint.scala

```diff
@@ -200,7 +200,8 @@ object ContentWidths {
 
   object MinuteMedia extends ContentRelation {
     override val inline = WidthsByBreakpoint(
-      mobile = Some(95.vw),
+      // In the minute inline images match the immersive header size
+      mobile = Some(1000.px),
```

Comment:
Makes sense now we spoke in person. Maybe the comment should elaborate more? I didn't know what the immersive header was.

If I understand correctly, we set this to a large pixel size because we use `object-fit` on the image.